src/System.Runtime.CompilerServices.VisualC/src/System.Runtime.CompilerServices.VisualC.csproj

```diff
@@ -4,18 +4,22 @@
   <PropertyGroup>
     <AssemblyName>System.Runtime.CompilerServices.VisualC</AssemblyName>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
   </PropertyGroup>
   <!-- Help VS understand available configurations -->
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net463_Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net463_Release|AnyCPU'" />
+  <!-- Shared CoreCLR and .NET Native -->
+  <ItemGroup Condition="'$(TargetGroup)' == ''">
+    <Compile Include="System\Runtime\CompilerServices\Attributes.cs" />
+  </ItemGroup>
   <ItemGroup>
     <TargetingPackReference Include="mscorlib" />
```

Comment:
done

---
In reply to: [80331452](https://github.com/dotnet/corefx/pull/11930#discussion_r80331452) [](ancestors = 80331452)